tidy_data/tidy_data.md

```diff
@@ -0,0 +1,737 @@
+
+<div class = "hint">
+This point is crucial enough that we want to make it stand out:
+
+**Uniting values is simpler than separating them.**
+
+It's impossible to tease out individual grades from an average, but if you have the individual grades, it's straightforward to calculate the average.  Names from two columns can be united by adding a space between them.  Values from a "systolic" and "diastolic" column can be united with a slash between them.  It is better to err on the side of being overly careful in separating out variables.
+
+</div>
+
+Another mistake that commonly occurs in human subjects research is the collection of race data with an assumption that a single value adequately captures race, such as "Asian", "Black", and so forth.  If a subject identifies with multiple races, "multiracial" could be added, for example.  But what if you want to study a genetic variant that occurs disproportionately in people of sub-sarahan African descent, and you're interested in all subjects with Black racial heritage, regardless of whether they identify with other races as well?  "Multiracial" does not answer your question about race, and unless you can track down your subject, there's no way to disentangle the wealth of information in the word "multiracial".
+
+<div class = "warning">
+An aside about race: in addition to making for more difficult use of race data, collecting data as a single value does not comply with federal standards.  Human subjects (or their representatives) should be able to endorse any race they identify as their own.  Therefore, it's much more helpful to begin right away with a True / False endorsement column for each race, so that you can identify easily who fits into various categories based on particular race identification, counts, or specific combinations:
+
+* White only
+* Indigenous and one other race
+* Non white, single race
+* Number of races 3 or higher
+* etc.
+
+An additional, important question that can be a source of disagreement in demographic and disparity research is what specific groups constitute races that can be endorsed.
+</div>
+
+### Quiz: Three Characteristics of Tidy Data
+
+
+<div class = "question">
+
+Which of the following are accurate statements about tidy data?
+
+[[X]] An observation is a single thing with measurable characteristics (like a mouse or a city)
+[[ ]] Observations should each be in their own column
+[[ ]] It's often a good idea to combine related data in a single cell, for instance, by encoding "4 year old female" as 4yf
+[[X]] "Height in meters" is an example of a variable.
+[[X]] Observations should each be in their own row
+[[?]] There are multiple correct answers!
+
+
+<div class = "answer">
+<details><summary>Click to see an explanation of the answer.</summary>
```

Comment:
liascript has built-in functionality to show explanations after a quiz is resolved; you don't need details tags. https://liascript.github.io/course/?https://raw.githubusercontent.com/liaScript/docs/master/README.md#70 You just need a row of at least 3 asterisks on the line immediately after the answers/hints, and then a corresponding set of at least 3 asterisks after the end of your explanation.

Reply:
I **think** the reason I went in this direction was to hide the answer until the learner wanted to click to expose it, but I'd rather stay in Liascript if possible.  I'll do a bit of experimentation!

Reply:
THIS IS SO MUCH BETTER thank you.  I have used my kludgy method all over the place so I will have some "fix it in production" changes tonight :/